### ISSUE TITLE
the problem with Clipspeak; 

### ISSUE THREAD

**Issue by Urryblind** [ruifontes/clipspeak#8]
Clipspeak;   2024.03.21; 
windows  10 x 64
Word 2007
pasting and cutting text in word does not work
the voiceover does not work and the insertion and cutting operation is blocked
